libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3492,6 +3492,9 @@ void GCS_MAVLINK::send_banner()
             send_text(MAV_SEVERITY_INFO, "%s", banner_msg);
         }
     }
+
+    // send smart_battery_info MAVLink message for each monitor 
+    send_smart_battery_info();
```

Comment:
There might be some push-back on this.

Reply:
Is there a better area/way to emit the message once at start-up or upon connection to GCS?  Still learning the code.

Reply:
Well, a better way might be for the GCS to probe for the relevant mavlink message and present it as appropriate.

ATM no GCS will handle the smart-battery-info message so....

Reply:
I added the smart_batery_info message to where it can be requested